app/_locales/en/messages.json

```diff
@@ -3008,11 +3008,16 @@
     "message": "Edit network details"
   },
   "nativeTokenScamWarningDescription": {
-    "message": "This network doesn't match its associated chain ID or name. Many popular tokens use the name $1, making it a target for scams. Scammers may trick you into sending them more valuable currency in return. Verify everything before you continue.",
-    "description": "$1 represents the currency name"
+    "message": "The network currency symbol entered does not match it’s associated currency symbol on the current chain ID. You have entered $1 while the detected currency symbol should be $2. Please verify you are connected to the correct chain. ",
```

Comment:
```suggestion
    "message": "The native token symbol does not match the expected symbol of the network with the associated chain ID. You have entered $1 while the expected token symbol is $2. Please verify you are connected to the correct chain.",
```

Reply:
@alfeng6 Can you verify that the copy @legobeat is suggesting is approved by Product? I was going off of your direction in the [Jira ticket](https://consensyssoftware.atlassian.net/browse/MMASSETS-384). Looks good to me.

Reply:
Copy looks good to me 👍